webapp/loggedin/modules/data/validationReport/validationReportView.js

```diff
@@ -0,0 +1,129 @@
+import './validationReportView.scss'
+
+import React from 'react'
+import { connect } from 'react-redux'
+
+import * as R from 'ramda'
+
+import TableView from '@webapp/loggedin/tableViews/tableView'
+import { useI18n, useOnUpdate } from '@webapp/commonComponents/hooks'
+
+import * as Survey from '@core/survey/survey'
+import * as Record from '@core/record/record'
+import * as NodeDef from '@core/survey/nodeDef'
+import * as Authorizer from '@core/auth/authorizer'
+
+import { appModuleUri, dataModules } from '@webapp/loggedin/appModules'
+
+import * as AppState from '@webapp/app/appState'
+import * as SurveyState from '@webapp/survey/surveyState'
+import * as RecordsState from '@webapp/loggedin/modules/data/records/recordsState'
+
+import { reloadListItems } from '@webapp/loggedin/tableViews/actions'
+
+import ValidationFieldMessages from '@webapp/commonComponents/validationFieldMessages'
+
+const validationReportModule = 'validationReport'
+
+const ValidationReportRowHeader = ({ nodeDefKeys }) => {
+  const i18n = useI18n()
+
+  return (
+    <>
+      <div>#</div>
+      <div>{i18n.t('common.path')}</div>
+      <div>{i18n.t('common.messages')}</div>
+      <div />
+    </>
+  )
+}
+
+const ValidationReportRow = ({ user, survey, row, idx, offset }) => {
```

Comment:
I would move this component in its own file

Reply:
Ok